test/Microsoft.AspNet.Mvc.Core.Test/UnitTestPrototype.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading;
+using Microsoft.AspNet.Http.Core.Collections;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core.Test
+{
+	public class UnitTestabilityTestPrototype
+	{
+		[Fact]
+		public void ControllerTestability_UsingContextRequestAbortedTest()
+		{
+			// Arrange
+			var controller = new TestabilityController();
+
+			var callback = new MvcControllerUnitTestHelperCallback()
+			{
+				OnRquestAborted = () => new CancellationToken(true)
+			};
+
+			// Act
+			new MvcControllerUnitTestHelper().Initialize(controller, callback);
```

Comment:
this will just be:

```c#
ControllerTestHelper.Create<TestabilityController)(callback);
```

or - 

```c#
ControllerTestHelper.Create<TestabilityController)(new ControllerTestCallback() // note the name change
    {
        On RequestAborted = () => new CancellationToken(true),
    });
```